apps/myaccount/src/components/multi-factor-authentication/authenticators/backup-code-authenticator.tsx

```diff
@@ -86,7 +101,9 @@ export const BackupCodeAuthenticator : FunctionComponent<BackupCodeProps> = (
     const [ remainingBackupCodes, setRemainingBackupCodes ] = useState<number>(0);
     const [ isCodesCopied, setIsCodesCopied ] = useState<boolean>(false);
     const [ isConfirmRegenerationModalOpen, setIsConfirmRegenerationModalOpen ] = useState<boolean>(false);
-    const [ isWarnRemaingBackupCodes, setIsWarnRemaingBackupCodes ] = useState<boolean>(false);
+    const [ isWarnRemainingBackupCodes, setIsWarnRemainingBackupCodes ] = useState<boolean>(false);
```

Comment:
It'd be better if we can make these variable names a little bit more descriptive IMO.

Reply:
addressed with [7f1090d](https://github.com/wso2/identity-apps/pull/3793/commits/7f1090d4101ffb73416166498cf4a6100d0cc16e)